src/app/components/snippet/snippet.component.ts

```diff
@@ -1,9 +1,9 @@
 import { Component, OnDestroy, OnInit } from '@angular/core';
 import { SelectSnapshot } from '@ngxs-labs/select-snapshot';
 import { Select } from '@ngxs/store';
+import { FileService } from '@snypy/rest-client';
 import { ResourceModel } from 'ngx-resource-factory/resource/resource-model';
```

Comment:
Please remove unused imports.

Reply:
Fixed. I also refactored some logic regarding the label loading.